packages/web-frontend/src/components/HierarchyItem.js

```diff
@@ -5,21 +5,6 @@
  * found in the LICENSE file in the root directory of this source tree.
  */
 
-/**
- * HierarchyItem
- *
- * Custom react hierarchy component that is composable enough for Tupaia.
- *
- * @prop {string} label The text to render in the hierarchyItem.
- * @prop {string} nestedMargin Amount of space an item should be nested relative to parent, any valid margin-left value.
- * @prop {array}  nestedItems An array of nested items to render as children. Will render iteractive expand arrow on left if provided.
- * @prop {boolean} isSelected True - render checked box on right; False - render unchecked box on right; null - render neither.
- * @prop {boolean} hasNestedItems Manually tell element to render left side arrow.
- * @prop {function} Icon Custom icon for the hierarchy item
- * All additional props go to material-ui FlatButton component.
- * @return {element} a HierarchyItem react component.
- */
```

Comment:
These were outdated docs and weren't useful during development - better to just look at the reasonably well named props imo